src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp

```diff
@@ -222,43 +224,50 @@ std::unique_ptr<ObjectData3D> TinyGltfImporter::doObject3D(UnsignedInt id) {
     CORRADE_INTERNAL_ASSERT(node.rotation.size() == 0 || node.rotation.size() == 4);
     CORRADE_INTERNAL_ASSERT(node.translation.size() == 0 || node.translation.size() == 3);
     CORRADE_INTERNAL_ASSERT(node.scale.size() == 0 || node.scale.size() == 3);
+    /* Ensure we have either a matrix or L-R-S */
```

Comment:
What does L stand for? (The glTF tutorial names it as TRS.)

Reply:
Location. (Like LocRotScale in blender)

Will change it to match the spec